cloudigrade/util/tests/helper.py

```diff
@@ -155,6 +155,58 @@ def generate_mock_ec2_instance(instance_id=None, image_id=None, subnet_id=None,
     return mock_instance
 
 
+def generate_mock_image(image_id=None, encrypted=False):
+    """
+    Generate a mocked EC2 Image object.
+
+    Args:
+        image_id (string): The AMI image id.
+
+    Returns:
+        Mock: A mock object with Image-like attributes.
+
+    """
+    root_device_name = '/dev/sda1'
+    root_device_type = 'ebs'
+    volume_types = ('gp2', 'io1', 'st1', 'sc1')
+    block_device_mappings = [
+        {
+            'DeviceName': root_device_name,
+            root_device_type.capitalize(): {
+                'Encrypted': encrypted,
+                'DeleteOnTermination': False,
+                'SnapshotId': generate_mock_snapshot_id(),
+                'VolumeSize': random.randint(0, 10),
+                'VolumeType': random.choice(volume_types)
+            }
+        }
+    ]
+
+    mock_image = Mock()
+    mock_image.image_id = image_id
+    mock_image.root_device_name = root_device_name
+    mock_image.root_device_type = root_device_type
+    mock_image.block_device_mappings = block_device_mappings
+    return mock_image
+
+
+def generate_mock_snapshot_id():
```

Comment:
I did something very similar in my branch! 😆  FWIW, my implementation was:

```
def generate_dummy_snapshot_id():
    """Generate a dummy AWS snapshot ID for testing purposes."""
    return 'snap-{}'.format(
        ''.join(random.choice(string.hexdigits[:16]) for _ in range(17))
    )
```
Although it's not strictly necessary, for more realistic random values, you should slice `[:16]` like I did or else it's twice as likely to get values a-f because `hexdigits` includes both lower and upper case letters.

Reply:
Yeah, on second though, let's just leave yours and I'll twiddle the merge in place when mine branch arrives. 👍